Artiom/src/main/java/Primitives.java

```diff
@@ -0,0 +1,14 @@
+//3. Write a java program to compare two strings lexicographically, ignoring case differences, if strings are different concatenate a given string to the end of another string.
+
+public class Primitives {
+
+    public static void main(String[] args) {
+        String lower_string = "string";
+        String upper_String = "String";
+        if (lower_string != upper_String) {
```

Comment:
Its not save to use == operator to compare strings
see also https://www.programiz.com/java-programming/examples/compare-strings#:~:text=You%20should%20not%20use%20%3D%3D,and%20not%20the%20object%20itself.

Reply:
done